examples/elasticsearch/es-rc.yaml

```diff
@@ -0,0 +1,49 @@
+apiVersion: v1
+kind: ReplicationController
+metadata:
+  name: es
+  namespace: default
```

Comment:
This should work in whatever namespace the user is currently switched into with `kubectl`. Please remove this line & update the `NAMESPACE` env var to use the downward API as described in https://github.com/kubernetes/kubernetes/pull/13025/files#r38514648.